src/main/java/com/backbase/ct/bbfuel/client/productsummary/ProductSummaryPresentationRestClient.java

```diff
@@ -83,21 +82,6 @@ public Response getProductSummaryArrangements() {
             .as(ArrangementsByBusinessFunctionGetResponseBody[].class));
     }
 
-    public List<ArrangementsByBusinessFunctionGetResponseBody> getUsForeignWireArrangements() {
```

Comment:
I am removing US_FOREIGN_WIRE because, this PR is specific for ACH_DEBIT. 
US_FOREIGN_WIRE will be handled in next/coming PR's

Reply:
But I didn't get why you removed it

Reply:
Since US_FOREIGN_WIRE is recently added and blocking payment ingestion, I will approve this pr. Shared the walkthrough to Pranjal in private message, she will fix it soon.